filters/openpolicyagent/openpolicyagent.go

```diff
@@ -443,13 +464,19 @@ func (registry *OpenPolicyAgentRegistry) new(store storage.Store, configBytes []
 
 	var logger logging.Logger = &QuietLogger{target: logging.Get()}
 	logger = logger.WithFields(map[string]interface{}{"skipper-filter": filterName})
-	manager, err := plugins.New(configBytes, id, store, configLabelsInfo(*opaConfig), plugins.Logger(logger), registry.withTracingOptions(bundleName))
+
+	configHooks := hooks.New()
+	if registry.overridePeriodicPluginTriggers {
```

Comment:
I wonder if we cannot just always install the Hook and make it a no-op if no override should happen. 

Once we have it, limiting the maximum bundle size would be sth that also required the hook to override some config...

Reply:
I find this a bit implicit. for the bundle size we might create another hook and configure everything in the startup